### PR TITLE
Added missing conditions in the TC's

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Added two test cases for the missing conditions

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
@aselle , thanks for reviewing my other PRs, can you pls review this PR.

File: tensorflow/lite/python/interpreter_test.py [hunk 1]
```diff
@@ -113,6 +113,21 @@ def testInvokeBeforeReady(self):
                                  'Invoke called on model that is not ready'):
       interpreter.invoke()
 
+  def testInvalidModelFileContent(self):
+    with self.assertRaisesRegexp(
+        ValueError, '`model_path` or `model_content` must be specified.'):
+      interpreter_wrapper.Interpreter(model_path=None, model_content=None)
+
+  def testInvalidindex(self):
```
Comment (jdduke) [inline tensorflow/lite/python/interpreter_test.py]: Nit: testInvalidIndex

File: tensorflow/lite/python/interpreter_test.py [hunk 2]
```diff
@@ -113,6 +113,21 @@ def testInvokeBeforeReady(self):
...
+    interpreter = interpreter_wrapper.Interpreter(
+        model_path=resource_loader.get_path_to_datafile(
+            'testdata/permute_float.tflite'))
+    interpreter.allocate_tensors()
+    #Invalid tensor index passed
```
Comment (jdduke) [inline tensorflow/lite/python/interpreter_test.py]: # Invalid tensor index passed.

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/python/interpreter_test.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/python/interpreter_test.py]: @jdduke , thanks for the review, i have updated the code as per suggestion. Kindly check.

Regards
Amit

File: tensorflow/lite/python/interpreter_test.py (hunk 2 above)
Comment (amitsrivastava78) [reply tensorflow/lite/python/interpreter_test.py]: @jdduke , thanks for the review, i have updated the code as per suggestion. Kindly check.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@jdduke , thanks for the review, i have updated the code as per suggestion. Kindly check.

Regards
Amit

**Review by jdduke — APPROVED** [review]

**Pull request merged**